e2e/support/helpers/e2e-ui-elements-helpers.js

```diff
@@ -2,7 +2,6 @@
 export const POPOVER_ELEMENT = ".popover[data-state~='visible']";
 
 export function popover() {
-  cy.get(POPOVER_ELEMENT).should("be.visible");
```

Comment:
This check made it difficult to check when the popover was closed, since `popover().should("not.exist")` would fail as the `popover()` function call would fail due to this assertion.

I'm not sure why this was here in the first place, it seems wrong for the getter function to assert if the ui element is visible, that should be the responsibility of the test itself. Also, none of the other getters in this file have this assertion.

Reply:
I'm going to tag @nemanjaglumac on this change. I feel like that must have been there for a reason.

Reply:
@nemanjaglumac Makes sense, I can keep it then

Reply:
@EmmadUsmani the thing you're trying to achieve is:
```js
cy.get(POPOVER_ELEMENT).should("not.exist");
```

https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/native/reproductions/18418-saved-question-db-appears-in-db-picker.cy.spec.js#L35

I think this is safer thing to do, given the amount of craziness we've seen historically with this component.